DOC-359: Updated cordapp-build-systems OS and Enterprise v 4.6 and 4.7 (#432)
corda/corda-docs: content/en/docs/corda-enterprise/4.6/cordapps/cordapp-build-systems.md

Signing the CorDapp enables its contract classes to use signature constraints instead of other types of the constraints,
for constraints explanation refer to [Contract Constraints](api-contract-constraints.md).
By default, the JAR file is signed by Corda development certificate.

> {{< alert >}}

Reviewer: ```suggestion
{{< warning >}}
```

> Confidential identities are signed with development keys by default. You will need to self-sign the '.jar' file when you deploy it to production.

Reviewer: ```suggestion
Confidential identities are signed with development keys by default. You will need to self-sign the `.jar` file when you deploy it to production.
```

> {{< /alert >}}

Reviewer: ```suggestion
{{< /warning >}}
```
